build.rs

```diff
@@ -109,6 +110,11 @@ const RING_SRCS: &[(&[&str], &str)] = &[
     (&[AARCH64], "crypto/fipsmodule/modes/asm/ghash-neon-armv8.pl"),
     (&[AARCH64], "crypto/poly1305/asm/poly1305-armv8.pl"),
     (&[AARCH64], SHA512_ARMV8),
+
+    (&[MIPS64], "crypto/fipsmodule/bn/asm/mips-mont.pl"),
+    (&[MIPS64], "crypto/poly1305/asm/poly1305-mips.pl"),
```

Comment:
Let's drop these two assembly modules.

I don't have time right now to review  the MIPS assembly code. mips-mont is no longer necessary to get things working because we now have Rust/C code that does the same thing. The C/Rust code isn't as fast as the assembly code, but we can make it faster than it is and maybe even faster than the assembly code, with not too much effort.

For poly1305-mips.pl, we don't have C/Rust code for poly1305 in *ring* yet, but we can get C code from BoringSSL, and that's what we should do. Doing that would be more time-efficient than trying to maintain the poly1305-mips code.